code/modules/clothing/head/misc.dm

```diff
@@ -44,13 +47,16 @@
 	permeability_coefficient = 0.01
 	siemens_coefficient = 0.9
 	body_parts_covered = 0
+	style_coverage = COVERS_WHOLE_HEAD
 
 /obj/item/clothing/head/hasturhood
 	name = "hastur's hood"
 	desc = "It's unspeakably stylish"
 	icon_state = "hasturhood"
 	flags_inv = BLOCKHAIR
 	body_parts_covered = HEAD|FACE|EYES
+	style = STYLE_HATHATHAT
+	style_coverage = COVERS_WHOLE_FACE
```

Comment:
Why `body_parts_covered` can't do the job of `style_coverage`?
Or simply style items covering entire zone they are worn in, it's not the armor after all.

Reply:
it's about style, not armor exactly; things that are recognizably there on the human sprite should be taken into account.

Reply:
So why can't style item apply effect to entire "organ/zone"?
For example, preacher or commander cowboy hat applying style effect to the whole head, not just hair.
Since style "armor" isn't a conventional armor, it don't have to physically cover everything.

Reply:
because you, the player, can see that the hat does not cover those things.